src/docs/release-notes/R32/FullNotes.mdx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+## React Native Reusable Workflows
+
+-   Finished Re-architected of React Native auth workflow package to allow for greater control and flexibility from the application side.
```

Comment:
Can we rearrange  the statement?
`Finished re-architecting the React Native auth workflow package to allow for greater control and flexibility from the application side.`